Jenkinsfile

```diff
@@ -2,8 +2,8 @@
 
 // Test plugin compatibility to recent Jenkins LTS
 // Allow failing tests to retry execution
-buildPlugin(jenkinsVersions: [null, '2.150.1'],
-            findbugs: [run:true, archive:true, unstableTotalAll: '0'],
+buildPlugin(jenkinsVersions: [null, '2.60.3'],
```

Comment:
Please explain why the tested Jenkins version was reduced from the current LTS to an older LTS?

If you're concerned that the build should add testing of Jenkins 2.60.3, please add 2.60.3 as an additional version string to be tested.  I don't want to stop testing the most recent LTS with changes on this branch.